scripts/build_glossary_page.py

```diff
@@ -31,6 +35,6 @@ def all_together_now(source_json_path, glossary_file_path):
 
 if __name__ == "__main__":
     all_together_now(
-        source_json_path="../docs/term_tags/terms.json",
-        glossary_file_path="../docs/glossary.md",
+        source_json_path="../docs/docusaurus/docs/term_tags/terms.json",
+        glossary_file_path="../docs/docusaurus/docs/glossary.md",
```

Comment:
Should the changes to this file be reverted as well?

Reply:
No these are the correct paths to those files relative to the scripts directory. Some directories got moved around in #7227